common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerMsalController.java

```diff
@@ -133,22 +134,23 @@ public class BrokerMsalController extends BaseController {
 
     private ResultFuture<Bundle> mBrokerResultFuture;
     private final String mActiveBrokerPackageName;
-    private final BrokerOperationExecutor mBrokerOperationExecutor;
     private final HelloCache mHelloCache;
     private final IPlatformComponents mComponents;
     private final Context mApplicationContext;
 
+    // Can be set in unit tests, otherwise this will be null.
+    @Nullable
+    private final List<IIpcStrategy> ipcStrategies;
+
+    @VisibleForTesting
     public BrokerMsalController(@NonNull final Context applicationContext,
```

Comment:
probably we can create a separate constructor which does not take the strategy, instead of making it nullable.

Reply:
there's already one.